Electrometer/v4/_init.yaml

```diff
@@ -0,0 +1,11 @@
+# init configuration
+mode: 1
+fits_files_path: "~/develop/electrometerFitsFiles"
+range: -0.01
+integration_time: 0.01
+median_filter_active: false
+filter_active: true
```

Comment:
why is filter_active true, whereas the filters themselves are false?

Reply:
No particular reason.

Reply:
Make them all false then please, just to minimize any confusion